desktop/assets/texts/en-US/dialogs.ftl

```diff
@@ -1,2 +1,9 @@
 dialog-ok = OK
-dialog-cancel = Cancel
\ No newline at end of file
+dialog-cancel = Cancel
+
+picker-title-open-file = Open a single file
+picker-filter-supported = All Supported Files
+picker-filter-swf = SWF (*.swf)
+picker-filter-spl = FutureSplash Animator (*.spl)
+picker-filter-ruf = Ruffle Bundle (*.ruf)
```

Comment:
I wonder - what's the chance that these indeed can be translated in any meaningful way to something else than what they currently are? And even if so, do we want that?

Reply:
Did we just do the time warp?

Reply:
Sorry, I recreated the note right when you replied to it... Because it seems the GH UI is buggy - this was meant to also correspond to `SWF (.swf)` above...

Reply:
Hmm, to me, even "bundle" is near that territory of "jargon" words that are not meant to be messed with... Like "shader" for example.

Reply:
I would expect that in most languages they will be the same as in English, however I added entries for them too for the sake of consistency and in case some language will require a translation (e.g. by adding a "file" suffix to SWF because it would be ambiguous otherwise — I don't know, just guessing, but I would have to know every language to make sure it's not needed). Looking for instance at Inkscape, some words might be translatable ("bundle" perhaps?):

<image src="https://github.com/user-attachments/assets/eada8ea7-233a-498b-bae6-ddaf97a643fd" width="100"/>
<image src="https://github.com/user-attachments/assets/7c62ca26-e5af-4665-93d9-28fdb980cd1c" width="100"/>